src/libpisp/backend/tiling/crop_stage.cpp

```diff
@@ -13,7 +13,7 @@
 using namespace tiling;
 
 CropStage::CropStage(char const *name, Stage *upstream, Config const &config, int struct_offset)
-	: BasicStage(name, upstream->GetPipeline(), upstream, struct_offset), config_(config)
+	: BasicStage(name, upstream->GetPipeline(), upstream, struct_offset), config_(config), started_(false)
```

Comment:
Are we using the `started_` field? I wasn't sure...

Reply:
Ah no, I see we are using it! Please ignore me...

Reply:
Ah yes, the next commit removes it. That's probably what I remembered. Just getting confused!